src/imap_mag/util/TimeConversion.py

```diff
@@ -46,3 +46,12 @@ def convert_j2000ns_to_date(
                 + CONSTANTS.J2000_EPOCH_POSIX
             )
         ]
+
+    @staticmethod
+    def convert_met_to_date(
+        met: npt.ArrayLike,
+        reference_epoch: np.datetime64 = CONSTANTS.IMAP_EPOCH,
+    ) -> list[date]:
+        """Convert mission elapsed time (MET) to Python date."""
+        j2000ns = TimeConversion.convert_met_to_j2000ns(met, reference_epoch)
+        return TimeConversion.convert_j2000ns_to_date(j2000ns)
```

Comment:
Should we add a comment to this that it does not use SPICE so is not accurate enough for science? And I assume this does not handle the leap seconds thing (which means we get different results to the SDC??)